etc/links.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for elinks
+# Description: Text WWW browser
+# This file is overwritten after every install/update
+#
+# Persistent local customizations
+include links.local
+# Persistent global definitions
+# include globals.local
+
+blacklist /tmp/.X11-unix
```

Comment:
Why do we need to blacklist /tmp/.X11-unix?

Reply:
Thought it's a common practice for CLI-only programs.

Reply:
try
`x11 none`

Reply:
```
$ grep -l "blacklist /tmp/.X11-unix" *.profile
7z.profile
atool.profile
bsdtar.profile
cpio.profile
curl.profile
dnscrypt-proxy.profile
dnsmasq.profile
elinks.profile
exiftool.profile
file.profile
git.profile
gpg-agent.profile
gpg.profile
gzip.profile
highlight.profile
less.profile
lynx.profile
mediainfo.profile
mutt.profile
odt2txt.profile
pdftotext.profile
seahorse-daemon.profile
server.profile
ssh-agent.profile
strings.profile
tar.profile
tracker.profile
unbound.profile
unrar.profile
unzip.profile
w3m.profile
wget.profile
xzdec.profile
$ grep -l "x11 none" *.profile
clamav.profile
file.profile
```

Reply:
https://github.com/netblue30/firejail/pull/2734#discussion_r289604681